### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by OhHeyAlan** [bunkerity/bunkerweb#247] [Help]
**Description**
Concise description of what you're trying to do, the expected behavior and the current bug.

Trying to get it to work...

**How to reproduce**
Give steps on how to reproduce the bug (e.g. : commands, configs, tests, environment, version, ...).

docker-compose up
docker-compose version 1.29.2, build 5becea4c

Client: Docker Engine - Community
 Version:           20.10.12
 API version:       1.41
 Go version:        go1.16.12
 Git commit:        e91ed57
 Built:             Mon Dec 13 11:45:48 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.12
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.12
  Git commit:       459d0df
  Built:            Mon Dec 13 11:43:56 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.12
  GitCommit:        7b11cfaabd73bb80907dd23182b9347b4245eb5d
 runc:
  Version:          1.0.2
  GitCommit:        v1.0.2-0-g52b36a2
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
localhost:/home/containers/nginx# 

**Logs**
The logs generated by bunkerized-nginx. **DON'T FORGET TO REMOVE PRIVATE DATA LIKE IP ADDRESSES !**

Starting nginx_myservice_1 ... done
Recreating nginx_mybunkerized_1 ... done
Attaching to nginx_myservice_1, nginx_mybunkerized_1
mybunkerized_1  | [2022-01-19 08:33:01] entrypoint - INFO - starting bunkerized-nginx ...
mybunkerized_1  | [2022-01-19 08:33:01] entrypoint - INFO - configuring bunkerized-nginx ...
mybunkerized_1  | [!] ERROR - wrong permissions on /etc/letsencrypt
nginx_mybunkerized_1 exited with code 1

**Comment by HugooB** [comment]
The error clearly indicates what went wrong: `ERROR - wrong permissions on /etc/letsencrypt`. 
You should fix the permissions on the `etc/letsencrypt` folder:

```
mkdir letsencrypt
chown root:101 letsencrypt
chmod 770 letsencrypt
```
